static/app/data/platformCategories.tsx

```diff
@@ -67,6 +67,7 @@ export const backend: PlatformKey[] = [
   'go-http',
   'go-iris',
   'go-martini',
+  'go-negroni',
```

Comment:
would be great to have tests that make sure the .py and .tsx files don't go out of sync. I find it a matter of time for this to happen (it's easy to miss the comment at the top of this array